astropy/utils/data.py

```diff
@@ -18,6 +18,8 @@
 import sys
 import time
 import urllib
+import urllib.error
```

Comment:
When I was running the file with the `-t` option it needed these imports.

Reply:
Thanks for the comment; this is independent verification that #6668 is actually necessary (and that I'm not just imagining things).